### PR TITLE
Updating Contributing.md 

### PR DESCRIPTION
**Pull request by Nanu308**
Updating Contributing.md to be more exact

## About The Pull Request
Updates the contributing.md to include new information and guidelines.

## Why It's Good For The Game
Lets contributor and maintainers easily see guidelines etc. for contributing to the game.

## Changelog
No player facing changes

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -161,7 +169,95 @@ If you are porting features/tools from other codebases, you must give them credi
...
+* Attachment sprite changes
+* Tacticool equipment and gear. We’re retro-future (or cassette punk if you will).
+</details>
+
+### Mapping
+<details>
+* Nightmare inserts with ridiculous loot or ones that are out of place (don’t put snow on LV, for example)
+* Major lane changes or remaps of legacy maps
+* Major remaps without prior approval
+* Additional detailing that degrades arena space or hinders gameplay in any sort of way
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:237]:
```suggestion
* Nightmare inserts with ridiculous loot or ones that are out of place (don’t put snow on LV, for example)
* Additional detailing that degrades arena space or hinders gameplay in any sort of way
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -161,7 +169,95 @@ If you are porting features/tools from other codebases, you must give them credi
...
+* Anything on the public task-board
+* New Alien strains
+* New Marine and Alien tech options
+* Bay12 legacy feature removal (such as wizard backend, laser eyes, etc)
+* Map specific survivor loadouts
+* Bug fixes and inconsistency fixes
+</details>
+
+## What we don't want
+There are things, features, balances, projects, and all other sorts of stuff that we don’t want player contributors touching. Some things are reserved to the Maintainer Team alone. While this list isn’t comprehensive, it’ll give you a good idea of what you should keep your hands off of while working with our codebase. If you make a PR that does one, or some of these things it doesn’t mean it will be automatically denied (and sometimes doing something on the following list is a good thing), but it will very likely drastically reduce your chances of having the PR accepted and merged, so don’t say we didn’t warn you!
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md]:
```suggestion
## What not to work on
While this list isn’t comprehensive, it’ll give you a good idea of what you should keep your hands off of while working with our codebase.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -161,7 +169,95 @@ If you are porting features/tools from other codebases, you must give them credi
...
+* Removal of non-legacy features
+* Major overhauls of gameplay without prior approval
+* Adding additional factions, antagonists, or species
+* Again no additional species or races, even Arcturians
+* New cassette tapes
+* Vehicle removals
+* New jobs / roles
+* Changes to whitelist roles
+* Removal of jobs / roles
+* Major reworks of jobs / roles
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:252]:
```suggestion
* No additional species or races, even Arcturians
```

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	- [Introduction](#introduction)
 	- [Getting Started](#getting-started)
 	- [Meet the Team](#meet-the-team)
-		- [Head Developer and Project Manager](#head-developer-and-project-manager)
+		- [Head Maintainer and Project Manager](#head-maintainer-and-project-manager)
 		- [Maintainers](#maintainers)
 		- [Issue Managers](#issue-managers)
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:10]: add your new headers to the table of contents here

Comment (Nanu308) [reply .github/CONTRIBUTING.md:11]: done

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -161,7 +178,80 @@ If you are porting features/tools from other codebases, you must give them credi
...
+<details>
+* Nightmare inserts with ridiculous loot or ones that are out of place (don’t put snow on LV, for example)
+* Additional detailing that degrades arena space or hinders gameplay in any sort of way
+</details>
+
+### Coding
+<details>
+* Only direct changes to balance numbers on an PR (damage, recoil, health, armor, movespeed, etc), with nothing else to add to the game (without prior approval)
+* No additional species or races, even Arcturians
+</details>
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:247]: change all of these to dashes instead of stars

Comment (Nanu308) [reply .github/CONTRIBUTING.md]: Removed <details> and changed * to -

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -161,7 +178,68 @@ If you are porting features/tools from other codebases, you must give them credi
...
+- Additional HUD styles
+- Bug fixes and inconsistency fixes
+
+
+### Mapping
+- Nightmare inserts
+- Object placement quality of life improvements (such as widening hallways and combat lanes cluttered with props)
+- Extra map detailing (so long as it doesn’t negatively impact performance)
+- Removal of dead-ends or gameplay dead-space on existing maps
+- New maps-
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:203]:
```suggestion
- New maps*
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -137,7 +154,7 @@ There is no strict process when it comes to merging pull requests. Pull requests
...
 
-* While we have no issue helping contributors (and especially new contributors) bring reasonably sized contributions up to standards via the pull request review process, larger contributions are expected to pass a higher bar of completeness and code quality *before* you open a pull request. Maintainers may close such pull requests that are deemed to be substantially flawed. You should take some time to discuss with maintainers or other contributors on how to improve the changes.
+* While we have no issue helping contributors (and especially new contributors) bring reasonably sized contributions up to standards via the pull request review process, larger contributions are expected to pass a higher bar of completeness and code quality -before- you open a pull request. Maintainers may close such pull requests that are deemed to be substantially flawed. You should take some time to discuss with maintainers or other contributors on how to improve the changes.
```
Comment (Watermelon914) [inline .github/CONTRIBUTING.md:157]:
```suggestion
* While we have no issue helping contributors (and especially new contributors) bring reasonably sized contributions up to standards via the pull request review process, larger contributions are expected to pass a higher bar of completeness and code quality *before* you open a pull request. Maintainers may close such pull requests that are deemed to be substantially flawed. You should take some time to discuss with maintainers or other contributors on how to improve the changes.
```

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
there may be a few others, but these are the ones I've found

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (Nanu308) [reply .github/CONTRIBUTING.md]: fixed

File: .github/CONTRIBUTING.md (hunk 6 above)
Comment (Nanu308) [reply .github/CONTRIBUTING.md]: fixed

**Pull request merged**